### PR TITLE
Verify legacy

### PR DESCRIPTION
**Pull request by mixcas**
Run: `yarn`

Add a /legacy route to link up old legacy odies to your account
screen shot 2018-06-21 at 1 29 20 pm

Also added an integration for react-router and redux. This allow us to access the history thru the redux store or for example, dispatch `push` from actions

### CODE REVIEW

File: src/redux/actions/legacyOdieActions.js [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+export const verifySubdomain = (subdomain) => {
+  return dispatch => {
+    const firebase = getFirebase();
+
+    // Clean error message
+    dispatch({
+      type: 'UNSET_LEGACY_ERROR',
+    });
+
```
Comment (mikeyrayvon) [inline src/redux/actions/legacyOdieActions.js:35]: validateSubdomain() before sending to firebase

File: src/redux/actions/legacyOdieActions.js [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+}
+
+export const setSubdomain = (subdomain) => {
+  return {
+    type: 'SET_LEGACY_SUBDOMAIN',
+    subdomain,
+  }
+}
+
+export const verifySubdomain = (subdomain) => {
```
Comment (mikeyrayvon) [inline src/redux/actions/legacyOdieActions.js]: use more semantic function name

File: src/redux/actions/legacyOdieActions.js [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+      .catch( error => {
+        console.log(error);
+        dispatch({
+          type: 'SET_LEGACY_ERROR',
+        });
+      });
+  }
+}
+
+export const verifyDocUrl = (docUrl) => {
```
Comment (mikeyrayvon) [inline src/redux/actions/legacyOdieActions.js]: use more semantic function name

**Comment by mixcas** [comment]
Fixed, @mikeyrayvon ready t go

**Pull request closed without merging**